src/cryptonote_basic/miner.cpp

```diff
@@ -54,19 +52,22 @@
   #include <mach/mach_host.h>
   #include <AvailabilityMacros.h>
   #include <TargetConditionals.h>
-#endif
-
-#ifdef __FreeBSD__
-#include <devstat.h>
-#include <errno.h>
-#include <fcntl.h>
-#include <machine/apm_bios.h>
-#include <stdio.h>
-#include <sys/resource.h>
-#include <sys/sysctl.h>
-#include <sys/times.h>
-#include <sys/types.h>
-#include <unistd.h>
+#elif defined(__linux__)
+  #include <unistd.h>
+  #include <sys/resource.h>
+  #include <sys/times.h>
+  #include <time.h>
+#elif defined(__FreeBSD__)
+  #include <devstat.h>
+  #include <errno.h>
+  #include <fcntl.h>
+  #include <machine/apm_bios.h>
+  #include <stdio.h>
+  #include <sys/resource.h>
+  #include <sys/sysctl.h>
+  #include <sys/times.h>
+  #include <sys/types.h>
+  #include <unistd.h>
 #endif
```

Comment:
That part seems to be just moving stuff around.

Reply:
Yes and no. This way avoids including this stuff everywhere miner.h is included.

Reply:
This sounds like you moved it from the header to the source file, but it's within the source file (and not much further at that). Maybe it used to be moved that way, and that got changed later ?

Reply:
Looking back at my changes, I see that I moved unistd.h, sys/resource.h, sys/times.h and time.h from miner.h to miner.cpp. I then also changed the indentation so that it is consistent (at least within the file).

Reply:
I don't have any problem with moving includes from miner.h to miner.cpp if it can be done. I'm talking about th FreeBSD includes, which just got moved around and reindented, and appear to otherwise be 100% identical (I just checked again).
But hey, since this is a good PR otherwise, I might just overlook that if it doesn't get fixed...